vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/ApiSteps.java

```diff
@@ -141,7 +133,7 @@ public void setUpRequestHeaders(ExamplesTable headers)
     @When("I issue a HTTP $httpMethod request for a resource with the URL '$url'")
     public void whenIDoHttpRequest(HttpMethod httpMethod, String url) throws IOException
     {
-        executeHttpCallSafely(prepareHttpRequestBuilder(httpMethod, url));
+        httpRequestExecutor.executeHttpCall(httpMethod, url, null);
```

Comment:
either make last parameter `Optional` or create overloaded method without this last parameter